src/ducks/sharing/autosuggest.styl

```diff
@@ -0,0 +1,43 @@
+@import 'cozy-ui'
+
+.container
+    position relative
+
+.suggestionsContainer
+    position absolute
+    top 100%
+    margin-top 1px
+    z-index 500
```

Comment:
Oh shit lol, I accidentally left that in. Yes I'm totally using one of our variables instead.